CHANGELOG.md

```diff
@@ -45,6 +45,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [#458](https://github.com/babylonlabs-io/babylon/pull/458) Set `CosmosProvider` functions as public in
 `babylonclient` package
 
+### State Machine Breaking
```

Comment:
oh is it `State Machine Breaking`  ? Based on ofiginal pr I though we do not an upgrade.

Also please rebase on latest `release/v1.x`  as  `1.0.0-rc6`was released

Reply:
It is state machine breaking in genesis only

Reply:
ah ok so no upgrade needed with that one 👍 

The only comment is to move it to `## Unreleased` section (as after rebase it ended in `## v1.0.0-rc6`

Reply:
I think it is just an improvement @GAtom22

Reply:
updated the changelog